zomes/generic_expression/Cargo.toml

```diff
@@ -12,8 +12,8 @@ name = "generic_expression"
 derive_more = "0"
 serde = "1"
 serde_json = "1"
-hdk = "0.0.110"
+hdk = "0.0.115"
 lazy_static = "1"
 jsonschema-valid = "0.4.0"
 chrono = { version = "0.4", features = ["serde"] }
-hc_time_index = { git = "https://github.com/holochain-open-dev/holochain-time-index", branch = "main" }
+hc_time_index = { git = "https://github.com/kaichaosun/holochain-time-index", branch = "bump-hdk" }
```

Comment:
Waiting for this: https://github.com/holochain-open-dev/holochain-time-index/pull/7

Reply:
This is now merged @kaichaosun